dockerbuilder/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:14.04
+MAINTAINER svanoort <samvanoort@gmail.com>
+
+
+# Dependencies plus vim for convenience, wget if you're trying to get fancy
+RUN apt-get update \
+  && apt-get install -y make unzip devscripts debhelper rpm expect createrepo ruby maven openjdk-7-jre \
+  && apt-get install -y vim \
+#  && apt-get install -y wget \ 
```

Comment:
True, adding it.  This was based on mounting a checked-out copy as a volume, but best to support all options.